src/teammate

```diff
@@ -145,27 +145,30 @@ else
     if [ "$WIP" = "1" ]
     then
       PR_URL=$(hub pull-request -m "$PR_TITLE" -b $branch -l $branch,WIP)
+      if [ "$?" = "1" ]; then err "Could not open a pull request to $branch. Did you forget to use the -u or -b option?"; fi
       MESSAGE="I opened a WIP pull request. Can you give me an early feedback?"
     else
       PR_URL=$(hub pull-request -m "$PR_TITLE" -b $branch -l $branch)
+      if [ "$?" = "1" ]; then err "Could not open a pull request to $branch. Did you forget to use the -u or -b option?"; fi
       MESSAGE="Can you review my new pull request?"
     fi
   done
 fi
 
-success "Pull request(s) correctly syncronized"
+step "Pull request(s) correctly syncronized"
 
 # Send Notification
 if [ "$NOTIFY" = "1" ]
 then
   FORMATTER "$AUTHOR" "$MESSAGE" "$PR_URL" "$PR_TITLE"
-  curl -X POST \
+  curl -f -X POST \
     $NOTIFICATION_HOOK \
     -H 'Content-Type: application/json' \
     -H 'cache-control: no-cache' \
     -d "$NOTIFICATION"
 
-  success "Slack notification sent!"
+  if [ "$?" != "0" ]; then err "Could not send notification to slack. You should check your config file."; fi
+  step "Slack notification sent!"
```

Comment:
Great job!
Could you remove "slack " from the step text? It is supposed to be generic and it was my bad putting it there at the first time 🤔